templates/vue/src/App.vue

```diff
@@ -69,8 +69,10 @@ export default {
           that.loggedIn = data["wp-auth-check"]
         })
       })
+      Promise.resolve(client.getAvatar()).then(savedAvatar => {
+        this.addAvatar(savedAvatar.data)
+      })
```

Comment:
I"m not sure if this check should be under app.vue, maybe under tapestryApp.vue?

Reply:
Why can't we just get this along with all the other user progress stuff so we don't have to do a separate request just for this. Is there a reason for this?

Reply:
Leaving as is as per the Zoom chat.